README.md

```diff
@@ -6,10 +6,11 @@ SQLDelight generates typesafe APIs from your SQL statements. It compile-time ver
 Example
 -------
 
-To use SQLDelight, apply the [gradle plugin](https://github.com/square/sqldelight#gradle) and put your SQL statements in a `.sq` file, like
-`src/main/sqldelight/com/example/HockeyPlayer.sq`. Typically the first statement creates a table.
+To use SQLDelight, apply the [gradle plugin](https://github.com/square/sqldelight#gradle) and put your SQL statements in a `.sq` file in `src/main/sqldelight`.  Typically the first statement in the SQL file creates a table.
```

Comment:
Should we also explicitly mention that you should use a folder/package structure as well?

Reply:
Yeah.  Including where the generated source ends up for people that aren't totally sure how that "magic" works.

I think that is a good sub-section to have, but not critical for providing your "quick start guide" where you just need:
* dependencies
* gradle plugin
* where to put `.sq` files
* how to use the generated code in Android/JVM/etc

Reply:
Reading over the docs again, and they are awesome for Kotlin Multiplatform, but I think a "quick start" for Android would suck people in, and the multi-platform would become a later benefit.

All good things to discuss, but not the intent of this PR.  Just the small change.  Maybe can create an issue for this follow up README work.

Reply:
yea id be very open to having more targeted getting started guides but keeping the readme more general

Reply:
When/if I have some time I might be able to blog something, but not for at least a week.  Sounds like a good plan.  Appreciate all the thought/time put into the library.  Hopefully can get some good use out of it and can contribute back with some posts about it.

Reply:
Success story: https://twitter.com/HandstandSam/status/1093554347877765122

Reply:
🎉